crates/litesvm/src/lib.rs

```diff
@@ -1234,11 +1235,27 @@ impl LiteSVM {
     pub fn expire_blockhash(&mut self) {
         self.latest_blockhash = create_blockhash(&self.latest_blockhash.to_bytes());
         #[allow(deprecated)]
-        self.set_sysvar(&RecentBlockhashes::from_iter([IterItem(
-            0,
-            &self.latest_blockhash,
-            self.fee_structure.lamports_per_signature,
-        )]));
+        {
+            let blockhashes = self.get_sysvar::<RecentBlockhashes>();
+            let mut entries = vec![];
```

Comment:
Is it possible to init with capacity, something like this:
```rust
let max_entries_len = blockhashes.len().min(MAX_RECENT_BLOCKHASHES);
let mut entries = Vec::with_capacity(max_entries_len);
```

Reply:
We should initialise it in LiteSVM::new and store it as a field on the LiteSVM struct

Reply:
@Aursen @kevinheavey, I've implemented Aursen's suggested change and run `cargo fmt`. I'm not quite understanding Kevin's suggestion - is that something either of you will have time to knock out?